src/index.html

```diff
@@ -11,6 +11,15 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="container">
+      <form class="big-bar" data-qa="big">
+        <input class="big-bar__search" type="search"
+          placeholder="Try “Los Angeles”" data-qa="keypress">
```

Comment:
Need to fix code style everywhere
```suggestion
        <input
          class="big-bar__search"
          type="search"
          placeholder="Try “Los Angeles”"
          data-qa="keypress"
        >
```

Reply:
Please, check demo link again ([DEMO LINK](https://alyatulupova.github.io/layout_search-bar-airbnb/), because this changes were done. The last version:
<img width="432" alt="Знімок екрана 2023-11-25 о 22 42 56" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/149272263/1d7940f9-3759-44bc-99c0-dcbb2806840b">